packages/node-integration-tests/utils/run-tests.ts

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable no-console */
+import childProcess from 'child_process';
+import os from 'os';
+
+const testPaths = childProcess.execSync('jest --listTests', { encoding: 'utf8' }).trim().split('\n');
+
+let testsSucceeded = true;
+const numTests = testPaths.length;
+const fails: string[] = [];
+
+const threads = os.cpus().map(async (_, i) => {
+  while (testPaths.length > 0) {
+    const testPath = testPaths.pop();
+    console.log(`(Worker ${i}) Running test "${testPath}"`);
+    await new Promise(resolve => {
+      const jestProcess = childProcess.spawn('jest', ['--runTestsByPath', testPath as string, '--forceExit']);
+
+      let output = '';
+
+      jestProcess.stdout.on('data', (data: Buffer) => {
+        output = output + data.toString();
+      });
+
+      jestProcess.stderr.on('data', (data: Buffer) => {
+        output = output + data.toString();
+      });
```

Comment:
Are you doing it this way (rather than just setting `stdio` to `inherit` in your `spawn` options) so that the output from each test is printed in a single block, rather than being interleaved with output from the other running tests? If so, a) good thinking!, and b) I think a comment saying so would be helpful.